spec/rails_helper.rb

```diff
@@ -9,6 +9,10 @@
 require 'rspec/rails'
 require 'support/factory_bot'
 require 'capybara/rails'
+
+Capybara.app_host = 'http://localhost:3000'
+Capybara.server_host = 'localhost'
+Capybara.server_port = '3000'
```

Comment:
I'm not quite sure why we suddenly need this. I think that you can safely remove this setup.